src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -31,6 +26,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static java.util.Locale.ROOT;
```

Comment:
I think we usually use `ENGLISH` in our code. Might make sense to use the same unless you have some special reason.